.github/workflows/land-blocking.yml

```diff
@@ -89,6 +89,19 @@ jobs:
               // Post comment on PR then fail this workflow
               should_fail = true;
             }
+            // Add repro cmd to message
+            try {
+              let env_vars = process.env;
+              body += "\nRepro cmd:\n";
+              body += `
+                $(aws ecr get-login --no-include-email --region us-west-2)
+                ./scripts/cti --tag dev_${env_vars.LIBRA_GIT_REV} --run bench
+              `;
+            } catch (err) {
+              if (err.code === 'ReferenceError') {
+                console.error("env var $LIBRA_GIT_REV is not set.");
+              }
+            }
```

Comment:
Should we include some notion of error to `body`? so that we know that something is not right?
Also, looks like current code simply ignores error `if err.code != 'ReferenceError'`, thats probably not what we desire

Reply:
hm... this actually is a good idea. Will patch it up.